Libplanet/Net/Messages/Message.cs

```diff
@@ -10,11 +10,20 @@
 
 namespace Libplanet.Net.Messages
 {
-    internal abstract class Message
+    /// <summary>
+    /// Abstract message class used in <see cref="ITransport"/>.
```

Comment:
How about this? I referenced `Libplanet.Tx.InvalidTxException`.

https://github.com/planetarium/libplanet/blob/25994e2d2c49ec39e39c8d66b80428827bc82ab3/Libplanet/Tx/InvalidTxException.cs#L6-L10

```suggestion
    /// Serves as the base class for messages used in <see cref="ITransport"/>.
```